### ISSUE TITLE
`twarc version` shouldn't require Twitter credentials

### ISSUE THREAD

**Issue by dunn** [DocNow/twarc#120]
As of 1.0.0, `twarc version` prompts for Twitter credentials before even printing the version:
```bash
$ twarc version
Please enter Twitter authentication credentials
consumer key:
```
It would be nice if you could run `version` prior to configuring twarc.

**Comment by edsu** [comment]
Thanks! I noticed this just today myself.

**Comment by edsu** [comment]
This was just released as v1.0.1 😺

**Comment by dunn** [comment]
That was fast, thanks!!
